tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,223 @@
+import logging
+import time
+import json
+import pytest
+
+import ocs.defaults as defaults
+import ocs.exceptions as ex
+from ocsci import tier1, ManageTest
+from ocs.ocp import get_ceph_tools_pod
+from utility.utils import run_cmd
+from utility import templating
+from ocs.utils import create_oc_resource
+from ocs import pod, ocp
+
+log = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+# Project name
+PROJECT_NAME = "ocs-372"
+
+PRJ = ocp.OCP(kind='Project')
+
+# yaml path
+TEMPLATES_DIR = "/tmp/"
+
+template_data = {}
+
+# Service
+OCP = ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Cephblockpool
+CBP = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Storage class
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Secret
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+
+# PVC
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=PROJECT_NAME
+)
+
+# Pod
+POD = ocp.OCP(
+    kind='Pod', namespace=PROJECT_NAME
+)
+
+
+def create_cephblock_pool(pool_name):
+
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'cephblockpool.yaml', TEMPLATES_DIR, _templating, template_data
+    )
+
+    # Check CephBlockPool is created
+    name = get_ceph_tools_pod()
+    po = pod.Pod(name, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    cmd = "ceph osd pool ls"
+
+    out, _, _ = po.exec_command(cmd=cmd, timeout=20)
+    assert pool_name in out
+
+
+def create_storageclass(sc_name, pool_name):
+
```

Comment:
missing docstring. 
Please apply to all functions

Reply:
done